bindings/pyroot_experimental/PyROOT/test/tfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+
+import ROOT
+from libcppyy import SetOwnership
+
+
+class TFileUnits(unittest.TestCase):
+    """
+    Test for the pythonizations of TFile
```

Comment:
I think it is also nice if the documentation of the test explains exactly what pythonization we are testing, e.g. [here](https://github.com/root-project/root/pull/2476/commits/3a3941758449207c0ca8323700d4856129ab5723).

Reply:
Sure.